BrowserKit/Sources/ComponentLibrary/BottomSheet/OnboardingBottomSheetViewController.swift

```diff
@@ -31,4 +40,36 @@ public class OnboardingBottomSheetViewController: BottomSheetViewController {
         // all presentation animations in viewWillAppear. This prevents duplicate
         // animations that could cause visual glitches in onboarding flows.
     }
+
+    // MARK: - Private Methods
+
+    private func setupIPadConstraints() {
+        guard UIDevice.current.userInterfaceIdiom == .pad else { return }
+
+        // Wait for the view hierarchy to be set up, then modify constraints
+        DispatchQueue.main.async { [weak self] in
+            self?.configureIPadLayout()
+        }
+    }
+
+    private func configureIPadLayout() {
```

Comment:
Took this opportunity to fix https://mozilla-hub.atlassian.net/browse/FXIOS-13733